### PR TITLE
Add v-charts w/ npm auto-update

### PR DESCRIPTION
**Pull request by lizheming**

### CODE REVIEW

**Comment by lizheming** [comment]
It has a little strange for CI action. It throw error while `index.js` file already can be found in the latest package version.

image

**Comment by MattIPv4** [comment]
`index.js` is not the same as `lib/index.js`, you're pulling files out of `lib` and into the root directory, so your filename also needs to reflect that and just be `index.js`

**Comment by lizheming** [comment]
@MattIPv4 Thank you for your mention, I missed `filename` property. I fixed it then it's ok now.

File: packages/v/v-charts.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          "sankey.js",
+          "sankey.min.js",
+          "scatter.js",
+          "scatter.min.js",
+          "style.css",
+          "style.min.css",
+          "tree.js",
+          "tree.min.js",
+          "waterfall.js",
+          "waterfall.min.js"
```
Comment (MattIPv4) [inline packages/v/v-charts.json:61]:
```suggestion
          "*.js"
```
We can simplify this down to a single line glob

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/v/v-charts.json (hunk 1 above)
Comment (lizheming) [reply packages/v/v-charts.json:61]: OK! But `*.js` will also include files like `*.common.js` and `utils.js` which used in CommonJS environment in the `lib` folder. maybe we need add another disallow rules? 

```
"*.js",
"!*.common.js",
"!utils.js"
```

Comment (MattIPv4) [reply packages/v/v-charts.json:61]:
```suggestion
          "lib/!(utils|*.common)?(.min).js"
```
Ah, didn't realise that the commonjs stuff was also bundled. This glob should get just the files you need.

[Glob test](https://www.digitalocean.com/community/tools/glob#?comments=true&glob=lib%2F%21%28utils%7C%2A.common%29%3F%28.min%29.js&matches=false&tests=%2F%2F%20Imported%20from%20v-charts%20NPM%20package&tests=package.json&tests=.editorconfig&tests=.eslintignore&tests=CHANGELOG_CN.md&tests=CHANGELOG.md&tests=CONTRIBUTING_CN.md&tests=CONTRIBUTING.md&tests=LICENSE&tests=README_CN.md&tests=README.md&tests=.github%2FISSUE_TEMPLATE.md&tests=.github%2FPULL_REQUEST_TEMPLATE.md&tests=lib%2Famap.common.js&tests=lib%2Famap.common.min.js&tests=lib%2Famap.js&tests=lib%2Famap.min.js&tests=lib%2Fbar.common.js&tests=lib%2Fbar.common.min.js&tests=lib%2Fbar.js&tests=lib%2Fbar.min.js&tests=lib%2Fbmap.common.js&tests=lib%2Fbmap.common.min.js&tests=lib%2Fbmap.js&tests=lib%2Fbmap.min.js&tests=lib%2Fcandle.common.js&tests=lib%2Fcandle.common.min.js&tests=lib%2Fcandle.js&tests=lib%2Fcandle.min.js&tests=lib%2Fchart.common.js&tests=lib%2Fchart.common.min.js&tests=lib%2Fchart.js&tests=lib%2Fchart.min.js&tests=lib%2Fconstants.js&tests=lib%2Fcore.js&tests=lib%2Ffunnel.common.js&tests=lib%2Ffunnel.common.min.js&tests=lib%2Ffunnel.js&tests=lib%2Ffunnel.min.js&tests=lib%2Fgauge.common.js&tests=lib%2Fgauge.common.min.js&tests=lib%2Fgauge.js&tests=lib%2Fgauge.min.js&tests=lib%2Fheatmap.common.js&tests=lib%2Fheatmap.common.min.js&tests=lib%2Fheatmap.js&tests=lib%2Fheatmap.min.js&tests=lib%2Fhistogram.common.js&tests=lib%2Fhistogram.common.min.js&tests=lib%2Fhistogram.js&tests=lib%2Fhistogram.min.js&tests=lib%2Findex.common.js&tests=lib%2Findex.common.min.js&tests=lib%2Findex.esm.js&tests=lib%2Findex.js&tests=lib%2Findex.min.js&tests=lib%2Fline.common.js&tests=lib%2Fline.common.min.js&tests=lib%2Fline.js&tests=lib%2Fline.min.js&tests=lib%2Fliquidfill.common.js&tests=lib%2Fliquidfill.common.min.js&tests=lib%2Fliquidfill.js&tests=lib%2Fliquidfill.min.js&tests=lib%2Fmap.common.js&tests=lib%2Fmap.common.min.js&tests=lib%2Fmap.js&tests=lib%2Fmap.min.js&tests=lib%2Fpie.common.js&tests=lib%2Fpie.common.min.js&tests=lib%2Fpie.js&tests=lib%2Fpie.min.js&tests=lib%2Fradar.common.js&tests=lib%2Fradar.common.min.js&tests=lib%2Fradar.js&tests=lib%2Fradar.min.js&tests=lib%2Fring.common.js&tests=lib%2Fring.common.min.js&tests=lib%2Fring.js&tests=lib%2Fring.min.js&tests=lib%2Fsankey.common.js&tests=lib%2Fsankey.common.min.js&tests=lib%2Fsankey.js&tests=lib%2Fsankey.min.js&tests=lib%2Fscatter.common.js&tests=lib%2Fscatter.common.min.js&tests=lib%2Fscatter.js&tests=lib%2Fscatter.min.js&tests=lib%2Fstyle.css&tests=lib%2Fstyle.min.css&tests=lib%2Ftree.common.js&tests=lib%2Ftree.common.min.js&tests=lib%2Ftree.js&tests=lib%2Ftree.min.js&tests=lib%2Futils.js&tests=lib%2Fwaterfall.common.js&tests=lib%2Fwaterfall.common.min.js&tests=lib%2Fwaterfall.js&tests=lib%2Fwaterfall.min.js&tests=lib%2Fwordcloud.common.js&tests=lib%2Fwordcloud.common.min.js&tests=lib%2Fwordcloud.js&tests=lib%2Fwordcloud.min.js&tests=src%2Fcomponent-list.js&tests=src%2Fcomponents%2Fdata-empty.vue&tests=src%2Fcomponents%2Floading.vue&tests=src%2Fconstants.js&tests=src%2Fcore.js&tests=src%2Findex.es.js&tests=src%2Fmodules%2Fanimation.js&tests=src%2Fmodules%2Fextend.js&tests=src%2Fmodules%2Fmark.js&tests=src%2Fpackages%2Famap%2Findex.js&tests=src%2Fpackages%2Famap%2Fmain.js&tests=src%2Fpackages%2Fbar%2Findex.js&tests=src%2Fpackages%2Fbar%2Fmain.js&tests=src%2Fpackages%2Fbmap%2Findex.js&tests=src%2Fpackages%2Fbmap%2Fmain.js&tests=src%2Fpackages%2Fcandle%2Findex.js&tests=src%2Fpackages%2Fcandle%2Fmain.js&tests=src%2Fpackages%2Fchart%2Findex.js&tests=src%2Fpackages%2Ffunnel%2Findex.js&tests=src%2Fpackages%2Ffunnel%2Fmain.js&tests=src%2Fpackages%2Fgauge%2Findex.js&tests=src%2Fpackages%2Fgauge%2Fmain.js&tests=src%2Fpackages%2Fheatmap%2Findex.js&tests=src%2Fpackages%2Fheatmap%2Fmain.js&tests=src%2Fpackages%2Fhistogram%2Findex.js&tests=src%2Fpackages%2Findex%2Findex.js&tests=src%2Fpackages%2Fline%2Findex.js&tests=src%2Fpackages%2Fline%2Fmain.js&tests=src%2Fpackages%2Fliquidfill%2Findex.js&tests=src%2Fpackages%2Fliquidfill%2Fmain.js&tests=src%2Fpackages%2Fmap%2Findex.js&tests=src%2Fpackages%2Fmap%2Fmain.js&tests=src%2Fpackages%2Fpie%2Findex.js&tests=src%2Fpackages%2Fpie%2Fmain.js&tests=src%2Fpackages%2Fradar%2Findex.js&tests=src%2Fpackages%2Fradar%2Fmain.js&tests=src%2Fpackages%2Fring%2Findex.js&tests=src%2Fpackages%2Fsankey%2Findex.js&tests=src%2Fpackages%2Fsankey%2Fmain.js&tests=src%2Fpackages%2Fscatter%2Findex.js&tests=src%2Fpackages%2Fscatter%2Fmain.js&tests=src%2Fpackages%2Ftree%2Findex.js&tests=src%2Fpackages%2Ftree%2Fmain.js&tests=src%2Fpackages%2Fwaterfall%2Findex.js&tests=src%2Fpackages%2Fwaterfall%2Fmain.js&tests=src%2Fpackages%2Fwordcloud%2Findex.js&tests=src%2Fpackages%2Fwordcloud%2Fmain.js&tests=src%2Futils.js)

**Review by MattIPv4 — APPROVED** [review]
Lgtm, ty!

**Pull request merged**